packages/pn-personafisica-webapp/src/component/Deleghe/DelegationsElements.tsx

```diff
@@ -23,13 +25,19 @@ export const Menu = (props: any) => {
   const { t } = useTranslation(['deleghe']);
 
   const handleOpenModalClick = () => {
+    if (props.menuType === 'delegates') {
+      trackEventByType(TrackEventType.DELEGATION_DELEGATE_REVOKE);
```

Comment:
Qui l'evento viene tracciato ogni volta che la modale viene aperta. Ipoteticamente io potrei aprire la modale e cliccare su annulla e riaprire la modale.
In questo modo avrei tracciato due eventi identici.
é il comportamento voluto o si vuole tracciare solo quando l'utente effettivamente fa la revoca o il reject?
Nel secondo caso vedere se gestire il tutto tramite status di redux